src/ChannelServer/World/Skill.cs

```diff
@@ -55,6 +55,11 @@ public int SpendSp
 			}
 		}
 
+		/// <summary>
+		/// Over Heat Counter
```

Comment:
Some of your comments could be a little more descriptive. From the name of the property, you can already infer that this is the "OverHeatCount(er)". The comment should ideally tell you more about what's going to happen if you use use the property. We weren't great at documenting the code either back then, but we should make an effort to improve on this =) Even a simple "Gets or sets the overheat count." tells you more about what's going to happen. You get or set that value, that's it, no hidden side-effects. Though you could even go into more detail about what that actually means.